[2022-03-18 21:53:54,398] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 21:53:54,470] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 21:53:54,471] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 21:53:54,471] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-03-18 21:53:54,473] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 21:53:54,506] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2022-03-17T00:00:00+00:00
[2022-03-18 21:53:54,521] {standard_task_runner.py:52} INFO - Started process 13192 to run task
[2022-03-18 21:53:54,530] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', '2022-03-17T00:00:00+00:00', '--job-id', '73', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpxemq748o', '--error-file', '/tmp/tmpmo2lv65r']
[2022-03-18 21:53:54,535] {standard_task_runner.py:77} INFO - Job 73: Subtask task_3
[2022-03-18 21:53:54,659] {logging_mixin.py:104} INFO - Running <TaskInstance: trigger_rule.task_3 2022-03-17T00:00:00+00:00 [running]> on host pc
[2022-03-18 21:53:54,855] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2022-03-18 21:53:54,857] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-03-18 21:53:54,858] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'exit 0']
[2022-03-18 21:53:54,872] {subprocess.py:75} INFO - Output:
[2022-03-18 21:53:54,873] {subprocess.py:83} INFO - Command exited with return code 0
[2022-03-18 21:53:54,924] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20220317T000000, start_date=20220318T162354, end_date=20220318T162354
[2022-03-18 21:53:54,991] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-18 21:53:55,023] {local_task_job.py:151} INFO - Task exited with return code 0
